src/PowerShellEditorServices.Hosting/Configuration/HostLogger.cs

```diff
@@ -15,9 +15,8 @@ namespace Microsoft.PowerShell.EditorServices.Hosting
     /// User-facing log level for editor services configuration.
     /// </summary>
     /// <remarks>
-    /// The underlying values of this enum attempt to align to both
-    /// <see cref="Microsoft.Extensions.Logging.LogLevel" /> and
-    /// <see cref="Serilog.Events.LogEventLevel" />.
+    /// The underlying values of this enum attempt to align to
+    /// <see cref="Microsoft.Extensions.Logging.LogLevel" />
     /// </remarks>
     public enum PsesLogLevel
```

Comment:
Since there are no longer two different enums to align with, can we just use MEL.LogLevel directly?

Reply:
It will cause the assembly to load before the ALC is ready, but we can make the Hostlogger match the same log levels. That's part of my other pr, this is just focused on removing serilog